jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/ResourceFactoryTest.java

```diff
@@ -339,104 +338,159 @@ public void testSplitSinglePath() throws URISyntaxException
     }
 
     @Test
-    public void testSplitIsAbsolute() throws Exception
+    public void testSplitIsAbsolute()
     {
         try (ResourceFactory.Closeable resourceFactory = ResourceFactory.closeable())
         {
-            String config = String.format("%s,%s,%s,%s,%s,%s", "dir/a", "foo/b", "bar/c", "jar:file:///foo/bar.jar!/", "file:///foo/bar.jar", "/foo/bar.jar");
-            List<Resource> resources = resourceFactory.split(config, ",", true);
-            resources.stream().forEach(r ->
+            List<String> rawInputs = new ArrayList<>();
+            rawInputs.add("dir/a");
+            rawInputs.add("foo/b");
+            rawInputs.add("bar/c");
+            if (OS.WINDOWS.isCurrentOs())
             {
-                //Note: do not test the value of absolute resolution of the relative references as this is system dependent.
-                assertThat(r, not(instanceOf(MountedPathResource.class))); //unwrapped so cannot be mounted
-                assertThat(r.getPath().isAbsolute(), is(true)); //must be absolute
-            });
+                rawInputs.add("jar:file:///C:/foo/bar.jar!/");
+                rawInputs.add("jar:file:///C:/foo/bar.jar");
+                rawInputs.add("C:/foo/bar.jar");
+            }
+            else
+            {
+                rawInputs.add("jar:file:///foo/bar.jar!/");
+                rawInputs.add("jar:file:///foo/bar.jar");
+                rawInputs.add("/foo/bar.jar");
+            }
 
-            //test that an unwrapped jar:file will try to be mounted and fail because it isn't a real location
-            resources = resourceFactory.split("jar:file:///foo/bar.jar!/", ",", false);
-            assertThat(resources.size(), is(0));
+            String rawConfig = String.join(",", rawInputs);
+            List<Resource> resources = resourceFactory.split(rawConfig, ",", true);
```

Comment:
The previous code explicitly tests `jar:file:///foo/bar.jar!/` with `unwrap` of  `false`. Here it is tested with `true`.

Reply:
We have the unwrap == false version in the other testcase.

Reply:
See: testSplitExistingJarFileSystemNotUnwrappedIsMounted